models/recruiters.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const recruiterModel = firestore.collection('recruiters')
+const userModel = firestore.collection('users')
+
+
+const addRecruiterInfo = async (recruiterData, username, timeStamp) => {
+  try {
+    const recruiterInfo = await recruiterModel.add(recruiterData)
+    const recruiter = await recruiterModel.doc(recruiterInfo.id).get()
+    const notifyUser= await sendDetails(recruiter, username, timeStamp)
+
+    return {
+      message: 'Request Submission Successful!!',
+      id: recruiterInfo.id
+    }   
+  } catch (err) {
+    logger.error('Error in adding recruiter', err)
+    throw err
+  }    
+}
+
+    
+const sendDetails= async(recruiter, username, timeStamp) =>{
+  try{
+    const user = await userModel.where('username', '==', username).limit(1).get()
+    var userName;
+    if(!user.empty)
+    {
+      user.forEach(doc => {
+        const userFname = doc.data().first_name
```

Comment:
Use ``camelCase`` instead of ``snake_case``
The consistency while writting the code is important.